### ISSUE TITLE
[A11y] Backend: Importers

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#3332]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Backend

### Admin user?

Yes

### Issue Description

Reference Link, Logged In: https://oregondigital.org/importers?locale=en

Ran the page using Accessibility Insights, recieved 90 errors:

<img width="669" alt="Image" src="https://github.com/user-attachments/assets/7f4a4962-b51c-4d2f-82a3-c0e20fcff9c9" />

### Suggested Fix

Suggested fix for the 90 items are the same, see below:

Fix the following:
Element is in tab order and does not have accessible text
Fix ONE of the following:
Element does not have text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Revalidate the page using Accessibility Insights.
